engine/src/java/com/runrev/android/Engine.java

```diff
@@ -2232,9 +2232,18 @@ else if (resultCode == Activity.RESULT_OK)
 					float t_width = t_bitmap.getWidth();
 					float t_height = t_bitmap.getHeight();
 					
-					InputStream t_exif_in = ((LiveCodeActivity)getContext()).getContentResolver().openInputStream(t_photo_uri);
-					
-					ExifInterface t_exif = new ExifInterface(t_exif_in);
+					/* In API Level >= 24, you have to use an input stream to make sure that access
```

Comment:
Hmm, using the existing InputStream `t_in` causes the captured pic to be rotated 90 degrees. I am wondering if this was the reason @montegoulding used a new one (the `t_exif_in` one) in the first place.

Reply:
Oh! Yes - the reason is that we've used the t_in stream above! Oops!

So below we should do:
```
if (Build.VERSION....)
{
    t_in.close();
    t_in = ((LiveCodeActivity)getContext()).getContentResolver().openInputStream(t_photo_uri);
}
```
This will close the (now finished) original stream, and then create a new one.